srcpkgs/xearth/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'xearth'
+pkgname=xearth
+version=1.1
+revision=1
+build_style=gnu-make
```

Comment:
this build_style doesn't exist, gnu-makefile

Reply:
Im confused right now, is it `gnu-make` or `gnu-makefile`